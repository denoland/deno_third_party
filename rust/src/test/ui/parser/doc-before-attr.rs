// compile-flags: -Z parse-only

/// hi
#[derive(Debug)] //~ERROR expected item after attributes
