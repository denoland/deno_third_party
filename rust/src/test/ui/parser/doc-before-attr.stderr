error: expected item after attributes
  --> $DIR/doc-before-attr.rs:4:16
   |
LL | #[derive(Debug)] //~ERROR expected item after attributes
   |                ^

error: aborting due to previous error

